openmdao/api.py

```diff
@@ -95,6 +95,9 @@
 # Units
 from openmdao.utils.units import convert_units, unit_conversion
 
+# Docs Util
```

Comment:
General comment, would this api be preferable:
    `om.Component.show_options_table()`
(or in long form:)
    `openmdao.core.component.Component.show_options_table()`
rather than:
    `om.show_options_table("openmdao.core.component.Component")`